bonus.js

```diff
@@ -0,0 +1,2 @@
+//Graag wil ik een uitleg erover a.u.b.
```

Comment:
Hier zijn de uitwerkingen te vinden: https://github.com/hogeschoolnovi/frontend-javascript-tech-it-easy-uitwerkingen